src/pages/edge/lifecycle-for-edge-network/migration-reference.md

```diff
@@ -0,0 +1,62 @@
+
+The session timeout configuration set in the Mobile Core card in Tags is not used in Lifecycle for Edge Network. There is no concept of a session or how long a user must background the application before a new Lifecycle event is dispatched. Instead, Lifecycle events are simply dispatched when the application is launched and when the application is closed.
+
+To understand this difference, take the following example. An application is using both Lifecycle for Edge Network and Lifecycle for Analytics. Mobile Core is configured with a session timeout of 300 seconds (5 minutes). The following chart displays when and how many events are dispatched from each path.
+
+### Lifecycle events Edge Network vs Analytics
+
+| **Application state** | **Edge Network event** | **Analytics event** |
+| :--- | :--- | :--- |
+| app launched | Application Launch (Foreground)  | Lifecycle Start and SessionInfo* |
+| app sent to background | Application Close (Background) | no event, session paused |
+| app launched after 3 minutes | Application Launch (Foreground) | no event, session resumed |
+| app sent to background | Application Close (Background) | no event, session paused |
+| app launched after 30 minutes | Application Launch (Foreground) | Lifecycle Start and SessionInfo* |
+
+<InlineAlert variant="info" slots="text"/>
+
+When using Lifecycle for Analytics, if the Analytics extension in Tags is configured with "backdate previous session info" and "offline enabled", then two events are dispatched on the start of a new session; one for the previous session info and one for the start of the new Lifecycle session. If these configurations are not set, then only one event is dispatched for the start of the new Lifecycle session.
+
+As you can see in the previous scenario, Lifecycle for Edge Network provides richer details on how a user interacts with an application. This level of granularity comes at a cost of more events being sent to the Edge Network than was previously sent when using Analytic directly.
+
+## Lifecycle data
+
+Lifecycle for Edge Network formats data to XDM, conforming to the [AEP Mobile Lifecycle Details](https://github.com/adobe/xdm/blob/master/docs/reference/adobe/experience/aep-mobile-lifecycle-details.schema.md) field group. The field group contains three subgroups: application details which are the Lifecycle data, environment details about the system software, and device details about the user's device. This Lifecycle data sent to the Edge Network is different in a couple key ways from the data sent from the mobile Analytics extension.
+
+1. Any metric or dimension which may be computed at reporting time from existing data is not included in Lifecycle for Edge Network events. This includes metrics such as number of launches, launches since upgrade, days since last use or launch, and dimensions such as events for daily or monthly user engagement, hour of day or day of week, and install date.
```

Comment:
it will be nice to bold italic the dimension/metric names. 

number of **_launches_**, **_launches since upgrade_**, **_days since last use_** or _**launch**_

also is it launch or launch number?